language/stdlib/modules/LibraAccount.move

```diff
@@ -562,6 +574,13 @@ module LibraAccount {
         );
         Event::publish_generator(&new_account);
         add_currencies_for_account<Token>(&new_account, add_all_currencies);
+        // VASPs start off by default with the full range of currencies. If a
```

Comment:
Yeesh, this parent/child limits problem is a nasty one...

Does this mean: VASPs start off with `LimitsDefinition`s for every currency even if they don't have a `Balance` resource for that currency? That feels a bit surprising to me; it would be nice if we can ensure that "balance exists in parent or child <=> limits exists in parent". I think we can do this by gating these calls with `add_all_currencies` (but not sure).

Reply:
Yea this is the one nasty part. 

We could have balance exists in VASP <=> limits in parent VASP. However, this means that if the parent VASP doesn't already have a published limits for a currency none of its children can have a balance in those currencies until it publishes it. e.g.
```
txn1: create_parent_vasp<Coin1>(...., add_all_currencies = false); ~> EXECUTED
txn2: create_child_vasp<Coin1>(parent,..., add_all_currencies = false); ~> EXECUTED
txn3: create_child_vasp<Coin2>(parent, ..., add_call_currencies = fasle); ~> ABORTED (since parent doesn't have limits for Coin2)
```

I'll take a look at this some more, hopefully this can be made cleaner without breaking things...

Reply:
I guess one could view "child can't have a balance in T until parent approves it" as a feature to prevent a child from (e.g.) accidentally violating capital controls... But this is nasty enough that there doesn't seem like a right answer, happy  with any approach you choose.